test-storybooks/portable-stories-kitchen-sink/nextjs/stories/__snapshots__/portable-stories.test.tsx.snap

```diff
@@ -794,7 +794,7 @@ exports[`renders nextHeaderStories stories renders Default 1`] = `
         </strong>
          
         <span>
-          firstName=Jane; lastName=Doe
+          firstName=Jane; ; lastName=Doe; 
```

Comment:
What's going on here?

Reply:
We are using now even closer what does Next does, so I guess it must be okay.